openrtb_ext/bidders.go

```diff
@@ -76,6 +76,7 @@ const (
 	Bidder33Across         BidderName = "33across"
 	BidderAcuityAds        BidderName = "acuityads"
 	BidderAdform           BidderName = "adform"
+	BidderAdformOpenRTB    BidderName = "adformOpenRTB"
```

Comment:
Should the bidder name also be lower case? Thoughts @hhhjort?

Reply:
There is only one exception to all lower case, probably best to not make it two.